server/pbench/bin/index-pbench

```diff
@@ -675,19 +678,23 @@ def get_samples(iteration, mdlog, tb):
         sample = l[2]
         if sample.startswith('sample'):
             samples.append(sample)
+    if len(samples) == 0:
+        samples.append('reference-result')
     return samples
 
 def get_pbench_hosts(iteration, sample, mdlog, tb):
     try:
         # N.B. Space-separated list
         return mdlog.get("tools", "hosts").split(' ')
-    except Exception:
+    except Exception as e:
```

Comment:
Catching `Exception` is very broad, what about catching [configparser.Error](https://docs.python.org/3/library/configparser.html#configparser.Error) instead?